pkg/iac/terraform/state/backend/http_reader.go

```diff
@@ -31,6 +33,18 @@ func NewHTTPReader(rawURL string, opts *Options) (*HTTPBackend, error) {
 		return nil, err
 	}
 
+	buf := new(bytes.Buffer)
+	_, err = buf.ReadFrom(res.Body)
+	if err != nil {
+		return nil, err
+	}
+
+	logrus.Tracef("HTTP(s) backend response body: %s", buf.String())
```

Comment:
Use something like this instead `logrus.WithFields(logrus.Fields{"response": buf.String()}).Trace("backend response body")`. It makes a more readable log record